src/main/java/seedu/moneygowhere/userinterface/ConsoleInterface.java

```diff
@@ -286,6 +291,31 @@ private void runCommandSortExpense(ConsoleCommandSortExpense commandSortExpense)
         printInformationalMessage(Messages.CONSOLE_MESSAGE_COMMAND_SORTED_EXPENSE_SUCCESS);
     }
 
+    private void runCommandAddTarget(ConsoleCommandAddTarget consoleCommandAddTarget) {
+        Target target = new Target(
+                consoleCommandAddTarget.getName(),
+                consoleCommandAddTarget.getDateTime(),
+                consoleCommandAddTarget.getDescription(),
+                consoleCommandAddTarget.getAmount(),
+                consoleCommandAddTarget.getCurrentAmount());
+        targetManager.addTarget(target);
+
+        DateTimeFormatter dateTimeFormat = DateTimeFormatter.ofPattern(
+                Configurations.CONSOLE_INTERFACE_DATE_TIME_OUTPUT_FORMAT
+        );
+        String targetStr = "";
+        targetStr += "Name          : " + target.getName() + "\n";
+        targetStr += "Date and Time : " + target.getDateTime().format(dateTimeFormat) + "\n";
+        targetStr += "Description   : " + target.getDescription() + "\n";
+        targetStr += "Amount        : " + target.getAmount() + "\n";
+        targetStr += "Current Amount: " + target.getCurrentAmount() + "\n";
+        printInformationalMessage(targetStr);
+
+        printInformationalMessage(Messages.CONSOLE_MESSAGE_COMMAND_ADD_TARGET_SUCCESS);
+
+        //saveToFile(targetManager.getTargets());
```

Comment:
Do consider using a Todo comment instead if a function is yet to be implemented.

Reply:
Oh shoot, alright I'll add that in! Thank you Fang Jie!